File: CS/CustomMessaging.Module.Win/Updater.cs
using System;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Data.Filtering;

namespace CustomMessaging.Module.Win {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            Person person1 = ObjectSpace.FindObject<Person>(
            CriteriaOperator.Parse("FirstName == 'Robert' && LastName == 'King'"));
            Person person2 = ObjectSpace.FindObject<Person>(
            CriteriaOperator.Parse("FirstName == 'Mary' && LastName == 'Tellitson'"));
            if (person1 == null) {
                person1 = ObjectSpace.CreateObject<Person>();
                person1.FirstName = "Robert";
                person1.LastName = "King";
                person1.Save();
            }
            if (person2 == null) {
                person2 = ObjectSpace.CreateObject<Person>();
                person2.FirstName = "Mary";
                person2.LastName = "Tellitson";
                person2.Save();
            }
        }
    }
}

File: VB/CustomMessaging.Module.Win/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Data.Filtering

Namespace CustomMessaging.Module.Win
    Public Class Updater
        Inherits ModuleUpdater
        Public Sub New(ByVal objectSpace As ObjectSpace, ByVal currentDBVersion As Version)
            MyBase.New(objectSpace, currentDBVersion)
        End Sub
        Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
            MyBase.UpdateDatabaseAfterUpdateSchema()
            Dim person1 As Person = ObjectSpace.FindObject(Of Person)(CriteriaOperator.Parse("FirstName == 'Robert' && LastName == 'King'"))
            Dim person2 As Person = ObjectSpace.FindObject(Of Person)(CriteriaOperator.Parse("FirstName == 'Mary' && LastName == 'Tellitson'"))
            If person1 Is Nothing Then
                person1 = ObjectSpace.CreateObject(Of Person)()
                person1.FirstName = "Robert"
                person1.LastName = "King"
                person1.Save()
            End If
            If person2 Is Nothing Then
                person2 = ObjectSpace.CreateObject(Of Person)()
                person2.FirstName = "Mary"
                person2.LastName = "Tellitson"
                person2.Save()
            End If
        End Sub
    End Class
End Namespace
